jsftemplating/src/main/resources/META-INF/faces-config.xml

```diff
@@ -19,7 +19,7 @@
 
 <faces-config xmlns="http://java.sun.com/xml/ns/javaee"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-facesconfig_1_2.xsd"
+        xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://www.oracle.com/webfolder/technetwork/jsc/xml/ns/javaee/web-facesconfig_1_2.xsd"
```

Comment:
This change should not have been necessary.  The java.sun.com URL should redirect to the new location.